src/engines/html5/html5.js

```diff
@@ -190,9 +195,26 @@ export default class Html5 extends FakeEventTarget implements IEngine {
       this._eventManager.listen(this._mediaSourceAdapter, CustomEvents.TEXT_TRACK_CHANGED, (event: FakeEvent) => this.dispatchEvent(event));
       this._eventManager.listen(this._mediaSourceAdapter, CustomEvents.ABR_MODE_CHANGED, (event: FakeEvent) => this.dispatchEvent(event));
       this._eventManager.listen(this._mediaSourceAdapter, CustomEvents.TEXT_CUE_CHANGED, (event: FakeEvent) => this.dispatchEvent(event));
+      this._eventManager.listen(this._mediaSourceAdapter, CustomEvents.ERROR, (event: FakeEvent) => this.dispatchEvent(event));
     }
   }
 
+  /**
+   * Handles errors from the video element
+   * @returns {void}
+   * @private
+   */
+  _handleVideoError(): void {
+    if (!this._el.error) return;
+    const errCode = this._el.error.code;
+    if (errCode == 1 /* MEDIA_ERR_ABORTED */) {
+      // Ignore this error code.js, which should only occur when navigating away or
```

Comment:
you removed the code that takes the browsers specific error path data, please return it